python/LAMBDA_CONCURRENCY_CHECK/LAMBDA_CONCURRENCY_CHECK.py

```diff
@@ -10,64 +10,65 @@
 # the specific language governing permissions and limitations under the License.
 
 '''
+Rule Name:
+  LAMBDA_CONCURRENCY_CHECK
 Description:
   Checks whether the AWS Lambda function is configured for function-level concurrent execution limit. The rule is NON_COMPLIANT if the Lambda function is not configured for function-level concurrent execution limit.
-

```

Comment:
Please maintain the Gherkin Formatting.
No need to remove these blank lines.
Please format the Gherkin as in below rule Gherkin:
https://github.com/awslabs/aws-config-rules/blob/master/python/ELASTICACHE_REDIS_CLUSTER_AUTO_BACKUP_CHECK/ELASTICACHE_REDIS_CLUSTER_AUTO_BACKUP_CHECK.py